Core 991 image sizing for variable loan card sizes

dyersituations opened the pull request:
https://kiva.atlassian.net/browse/CORE-991
https://kiva.atlassian.net/browse/CORE-1004

- Added support to new loan card for larger image based on "2 per row"
- Added the new card to all components used for lending-home

![image](https://user-images.githubusercontent.com/16867161/215658766-259ef4d9-e5a9-4843-98e1-22bdeafd3980.png)

![image](https://user-images.githubusercontent.com/16867161/215658829-7abe4d56-d68b-4175-af66-0a33cb1e5ee5.png)

dyersituations commented on src/components/LoanCollections/KivaClassicLoanCarousel.vue:
```diff
@@ -63,7 +72,16 @@
 				<template v-for="(loanId, index) in augmentedLoanIds">
 					<!-- show loan card -->
 					<!-- TODO Re-implement card position analytics -->
+					<kiva-classic-basic-loan-card-exp
```
We currently use this carousel in the lending-home page, and this work does not affect the new home page. If needed, I could create a copy of this carousel for the experiment, but I figured this was a minimal change to the component.

dyersituations replied:
Actually why don't I match what we do elsewhere in lending-home and use the `kv-carousel`.

mcstover replied:
Ok, that's fine, just didn't want to set the expectation that this new card would be available on the homepage.

dyersituations replied:
Yeah no worries. And yeah I'll change the carousel real quick, since it'll reduce the potential impact of the exp code.

dyersituations commented:
@roger-in-kiva @mcstover I updated the solution to use the existing 'LendingCategorySection`. Now all carousels are the same in lending-home, which is good. Also removed the "loop" functionality of `kv-carousel` in this usage, since there's a small styling bug with it when it loops.

roger-in-kiva commented on src/components/LoanFinding/PartnerSpotlightSection.vue:
```diff
@@ -1,35 +1,41 @@
 <template>
-	<div class="tw-w-full tw-bg-secondary">
+	<div v-if="mfiRecommendationsLoans.length > 0" class="tw-w-full tw-bg-secondary">
```
This condition was useful having this feat at the top of the lbc page; not sure if really needed here being at the bottom.

dyersituations replied:
It is needed because the loans load async, and the `LendingCategorySection` doesn't watch the `loans` prop. This was a simpler solution than adding a `watch`.
